webapp-django/crashstats/authentication/management/commands/auditgroups.py

```diff
@@ -42,10 +42,8 @@ class Command(BaseCommand):
     help = "Audits Django groups and removes inactive users."
 
     def add_arguments(self, parser):
-        # FIXME(willkg): change this to default to False after we've tested
-        # it.
         parser.add_argument(
-            "--persist", action="store_true", help="persists recommended changes to db"
+            "--dry-run", action="store_true", help="Whether or not to do a dry run."
```

Comment:
Oops--this should be a `store_false`. Bah.

Reply:
My bad--I copy-and-pasted this from the other command, so this is fine. Bah to the bah.